packages/terra-core-docs/src/terra-dev-site/doc/show-hide/About.1.doc.mdx

```diff
@@ -42,9 +46,113 @@ The Show-Hide component must be composed inside the [Base][1] component with a l
 ## Usage
 
 ```jsx
-import ShowHide from 'terra-show-hide';
+import ShowHide, { Focuser } from 'terra-show-hide';
 ```
 
+## Accessibility
```

Comment:
Wonder if this section would be better placed after the "Examples" section that explains the basic usage.

Reply:
Thank you, I am also not sure where to place that block. We might want to re-visit that question after the rest of documentation is approved.